tools/build_velox.sh

```diff
@@ -57,6 +58,7 @@ if [ $BUILD_VELOX_FROM_SOURCE == "ON" ]; then
 
     sed -i '/libprotobuf-dev/d' scripts/setup-ubuntu.sh
     sed -i '/protobuf-compiler/d' scripts/setup-ubuntu.sh
+    sed -i '/^sudo apt install/a\  libiberty-dev \\' scripts/setup-ubuntu.sh
```

Comment:
Is `libiberty-dev` required by Gluten or Velox, directly?

Reply:
Yes. I meet an error when compiling without this lib
And the document [Velex.md](https://github.com/oap-project/gluten/blob/main/docs/Velox.md) includes the lib as required static library